async-openai/src/types/vector_store.rs

```diff
@@ -18,8 +18,7 @@ pub struct CreateVectorStoreRequest {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub file_ids: Option<Vec<String>>,
     /// The name of the vector store.
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub name: Option<String>,
```

Comment:
Officially this is still optional, what's the reason for making it non-optional?

https://platform.openai.com/docs/api-reference/vector-stores/create#vector-stores-create-name

Reply:
Reverted in [next commit](https://github.com/64bit/async-openai/pull/230/commits/d368ce06fd470dec498bcb97863436b36b33416d)